contracts/tokenbridge/ethereum/L1AtomicTokenBridgeCreator.sol

```diff
@@ -127,19 +150,18 @@ contract L1AtomicTokenBridgeCreator is Initializable, OwnableUpgradeable {
      *      2 retryable tickets  to deploy L2 side. 1st one deploy L2 factory and 2nd calls function that deploys and inits
      *      all the rest of the contracts. L2 chain is determined by `inbox` parameter.
      */
-    function createTokenBridge(address inbox, uint256 maxGasForFactory, uint256 maxGasForContracts, uint256 gasPriceBid)
-        external
-        payable
-    {
+    function createTokenBridge(address inbox, uint256 maxGasForContracts, uint256 gasPriceBid) external payable {
```

Comment:
If somebody would try to deploy again for certain inbox it would fail due to re-used salt which results in same address, ie. in `_deployL1Contracts`: `address proxyAdmin = address(new ProxyAdmin{ salt: _getL1Salt(OrbitSalts.L1_PROXY_ADMIN, inbox) }());`

I guess we don't need explicit check then (which would require tracking used inboxes)?

Reply:
a comment explaining this would be nice

Reply:
done